packages/modelviewer.dev/src/docs-and-examples/create-html.ts

```diff
@@ -33,6 +33,43 @@ const CategoryConstant: Category = {
   Slots: [],
 }
 
+function getExamples(category: any):
+    string {
+      const examples = category['examples'];
+      let examplesString = '';
+      for (const example of examples) {
+        examplesString += `
+<h4 class="subcategory-header">
+  <a class="darken" href="#${example.htmlId}">${example.name}</a>
+</h4>`;
+      }
+      return examplesString;
+    }
+
+function createExampleSidebarCategory(category: any) {
+  const htmlName = category['htmlName'];
+  const windowHref = window.location.href;
+  const isActive = windowHref.indexOf(htmlName) >= 0 ? true : false;
+
+  const container = document.getElementById('sidebar-category-container');
+  container!.innerHTML += `
+  <div class="category">
+    <h3 id=${htmlName.concat('-sidebar')}>
+      <a class="darken" href="../${htmlName}">${category['name']}</a>
+    </h3>
+    <div class="subCategory">
+      ${isActive ? getExamples(category) : ''}
+    </div>
+  </div>
+    `;
+}
+
+export function createExamplesSidebar(json: any[]) {
+  for (const category of json) {
+    createExampleSidebarCategory(category);
```

Comment:
This function is trivial enough you might just combine them (assuming the inner one isn't used elsewhere).

Reply:
hmmm, I like the little bit of extra readability this does, and it is consistent with how I iterate through jsons in other places.

Reply:
My only trouble with the readability is that I can't easily tell which function is which because they only differ by one letter in the middle. But I agree, it's not a big deal.

Reply:
createExample**s**Sidebar vs createExampleSidebar**Category**?